test/cases/json/import-by-name/index.js

```diff
@@ -0,0 +1,21 @@
+import * as c from "../data/c.json";
+import * as d from "../data/d.json";
+import { bb, aa } from "../data/e.json";
+import f, { named } from "../data/f.json";
+import g, { named as gnamed } from "../data/g.json";
+
+it("should be possible to import json data", function() {
+	c[2].should.be.eql(3);
```

Comment:
I don't think numeric keys should be possible for a module namespace object since all exports need to be valid `IdentifierName`s. https://tc39.github.io/ecma262/#sec-exports

Not sure what a good way to check for that would be.

Reply:
Correct, those identifiers should not be accessible/added to the namespace exotic object. I would assume the same is true for non-identifier property keys as well.